__tests__/auth.test.ts

```diff
@@ -28,8 +28,13 @@ describe('Given an event with an [INCORRECT] scope', () => {
 
     const authInstance = new Auth();
 
-    expect(authInstance.checkScopesAndResolve(mockedEvent, ['incorrect scope'])).resolves.toThrow('Error: You are not authorized!');
```

Comment:
This just does not work. Javascript will not wait here but call the code and then jump to the next line and stop the jwksMock. After it has been stopped the request will be fired which will then not be matched by `nock`.